public/javascripts/SVLabel/src/SVLabel/onboarding/Onboarding.js

```diff
@@ -475,7 +475,15 @@ function Onboarding(svl, actionStack, audioEffect, compass, form, handAnimation,
         var mission = missions[0];
 
         missionContainer.setCurrentMission(mission);
-        modalMission.setMissionMessage(mission, neighborhood);
+        if (missionContainer.isTheFirstMission()) {
+            svl.initialMissionInstruction = new InitialMissionInstruction(svl.compass, svl.map,
+                svl.neighborhoodContainer, svl.popUpMessage, svl.taskContainer, svl.labelContainer);
+            modalMission.setMissionMessage(mission, neighborhood, null, function () {
+                svl.initialMissionInstruction.start(neighborhood);
+            });
+        }else{
+            modalMission.setMissionMessage(mission, neighborhood);
+        }
```

Comment:
I modified the if condition. The mission instructions pop-up after skipping the tutorial, retaking the tutorial, or finishing the regular tutorial. I can't recreate an instance in which the instructions will show up twice. Please let me know if you can recreate this issue in the latest code!